### PR TITLE
Add model

### PR DESCRIPTION
**Pull request by AashitaP**
- created data models with single description field
- one question -> many answers
- no other validation currently

### CODE REVIEW

File: app/models/answer.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class Answer < ApplicationRecord
+  belongs_to :question
```
Comment (icecubewang) [inline app/models/answer.rb:2]: Probably a good idea to add some constraints as well, such as no empty content, max length limits, etc.

File: app/models/question.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+class Question < ApplicationRecord
+  has_many :answers
```
Comment (icecubewang) [inline app/models/question.rb:2]: Same as above.

Comment (AashitaP) [reply app/models/question.rb:2]: I added length and presence, should I add more

Comment (icecubewang) [reply app/models/question.rb:2]: Should be good for now

**Review by icecubewang — APPROVED** [review]
LGTM. See if @LanghaoZ has more comments to add.

**Comment by icecubewang** [comment]
Just to point out that we'll probably need to add unit tests for the restrictions we just added (i.e. presence and length limit). This could be done in another PR when we have time. This one is good to go.

**Pull request merged**